build/Dockerfile

```diff
@@ -0,0 +1,50 @@
+ARG UBUNTU_VERSION=16.04
+
+# Use Ubuntu as the base image
+FROM ubuntu:${UBUNTU_VERSION}
+
+ARG GO_VERSION=1.23.2
+ARG ARCH='amd64'
+ARG GH_VERSION='2.61.0'
+
+# Install dependencies
+RUN apt-get update && apt-get install -y \
+    gnupg-agent \
+    unzip \
+    zip \
+    curl \
+    wget \
+    expect \
+    git \
+    tar \
+    gcc \
+    jq \
+    g++ \
+    gnupg2 \
+    gnupg-agent \
+    debsigs \
+    rpm \
+    build-essential \
+    software-properties-common \
+    python-software-properties \
+    gcc-arm-linux-gnueabi \
+    dpkg-sig \
+    gcc-aarch64-linux-gnu
+
+# Install Go
+RUN curl -sSL https://golang.org/dl/go${GO_VERSION}.linux-${ARCH}.tar.gz -o go${GO_VERSION}.linux-${ARCH}.tar.gz && \
+    tar -C /usr/local -xzf go${GO_VERSION}.linux-${ARCH}.tar.gz && \
+    rm go${GO_VERSION}.linux-${ARCH}.tar.gz
+
+# Set Go environment variables
+ENV PATH="/usr/local/go/bin:${PATH}"
```

Comment:
where ${PATH} is defined?

Reply:
It is updating the OS PATH variable after adding the Go bin path.

Reply:
yes, I am dumb